File: 2639-separate-the-digits-in-an-array/separate-the-digits-in-an-array.py
class Solution:
    def separateDigits(self, nums: List[int]) -> List[int]:
        r=[]
        for i in range(len(nums)-1,-1,-1):
            n=nums[i]
            while n:
                d=n%10
                n//=10
                r.append(d)
        r.reverse()
        return r

File: 2639-separate-the-digits-in-an-array/separate-the-digits-in-an-array.cs
public class Solution {
    public int[] SeparateDigits(int[] nums) {
        List<int> num = new List<int>();
        for(int i = nums.Length-1; i>=0; i--){
            int n = nums[i];
            while (n>0){
                int d = n%10;
                n/=10;
                num.Add(d);
            }
        }
        num.Reverse();
        return num.ToArray();
    }
}